modules/cluster-management/src/integrationTest/java/org/apache/ignite/internal/cluster/management/ItClusterManagerTest.java

```diff
@@ -290,6 +296,126 @@ void testInitInvalidNodesAsync() throws Exception {
         );
     }
 
+    @Test
+    void testNoConfigurationReordering() throws Exception {
+        startCluster(5);
+
+        ClusterManagementGroupManager clusterManager = cluster.get(0).clusterManager();
+
+        List<String> nodes = cluster.stream().map(MockNode::name).limit(3).collect(Collectors.toList());
```

Comment:
Let's import `Collectors.toList` statically so that it reads `collect(toList())`, close to plain English